CHANGELOG.md

```diff
@@ -6,8 +6,9 @@
 Added full support to Image Streamer Rest API version 300:
    - Build Plan
    - Deployment Plans
-   - Plan Scripts
+   - Golden Image
```

Comment:
Please fix here.

All the entries should be in plural or all should be in singular.